### PR TITLE
[LP0004] Updates based on review manager meeting

### PR DESCRIPTION
**Pull request by llvm-beanz**
This updates the proposal with changes discussed during the review manager meeting. The changes are:

* Update references to code owners to reflect the new maintainer role.
* Clarify that the project council will serve as a catch all for issues not covered by an area team.
* Explicitly state that area teams must communicate a timeline for decision making.
* Update voting contributor identification based on recent developer policy changes around email addresses.
* Add additional language around this document being living, and explicitly require the project council to annually review feedback and evolve the process.
* Include a timeline for initial adoption that includes testing voter identification scripting.

### CODE REVIEW

File: proposals/LP0004-project-governance.md
Comment (cyndyishida) [inline proposals/LP0004-project-governance.md:142]:
```suggestion
Any project area that has at least three members interested in forming an _area team_ can request the _project council_ form one. The _project council_ will then consider the needs of the project and determine whether to form a new team or not.
```

File: proposals/LP0004-project-governance.md
Comment (AaronBallman) [inline proposals/LP0004-project-governance.md:40]:
```suggestion
Each year, the _Project Council_ will collect feedback from _area teams_, _maintainers_, and the wider community on what is working, not working, or could be improved. That feedback will be discussed and this proposal will evolve to meet the needs of the community.
```

**Review by dwblaikie — APPROVED** [review]
(again, approving as incremental improvement of this draft, not as approval of the proposal)

Improvements look good to me - though please wait for at least one or two other approvals (@cyndyishida and @AaronBallman perhaps, though their suggestions so far seem small enough that they're easily validated without necessarily waiting on their individual confirmation specifically).

File: proposals/LP0004-project-governance.md
Comment (rnk) [inline proposals/LP0004-project-governance.md:40]: I would expand on "feedback will be discussed" to say something active, like "the project council will collect that feedback and present it at the developer meeting before each election cycle."

**Review by AaronBallman — APPROVED** [review]
LGTM!

File: proposals/LP0004-project-governance.md
Comment (rnk) [inline proposals/LP0004-project-governance.md:116]: To expand on facilitating decision making:

"Area teams should prepare a typical meeting agenda by collecting all the active RFCs in the community or significant disagreements in pull requests. By the end of the team meeting, the area team should try to identify actionable next steps or information to gather so the RFC can proceed."

File: proposals/LP0004-project-governance.md
Comment (rnk) [reply proposals/LP0004-project-governance.md:40]: We discussed dropping the mention of the developer meeting, which I think is fine. Just "shared ahead of the election cycle" seems good. We also used the words "transparency report" with possible operational changes.

File: proposals/LP0004-project-governance.md
Comment (echristo) [reply proposals/LP0004-project-governance.md:116]: +1 here to this.

**Review by echristo — APPROVED** [review]
LGTM. A couple of inline comments.

File: proposals/LP0004-project-governance.md
Comment (rnk) [reply proposals/LP0004-project-governance.md:116]: Looks good!

**Review by cyndyishida — APPROVED** [review]
LGTM

**Review by ldionne — APPROVED** [review]

**Review by tstellar — APPROVED** [review]

**Pull request merged**
